server/routes/category.js

```diff
@@ -4,6 +4,34 @@ const Category = db.Category;
 
 const router = express.Router();
 
+router.get('/', (req, res, next) => {
+    Category.findAll()
+    .then((categories) => res.json(categories))
+    .catch(next);
+});
 
+router.get('/:id', (req, res, next) => {
+    if(req.params.id){
```

Comment:
do we need to do this logic here?

Reply:
No, I removed it now.  Can look into error handling later.